tests/rule_based_profiler/parameter_builder/test_parameter_computations.py

```diff
@@ -250,92 +253,10 @@ def test_bootstrap_point_estimate_scipy_efficacy(
         )
         # Actual false-positives must be within the efficacy tolerance of desired (configured)
         # false_positive_rate parameter value.
-        assert (
-            false_positive_rate - EFFICACY_TOLERANCE
-            <= actual_false_positive_rates[distribution]
-            <= false_positive_rate + EFFICACY_TOLERANCE
-        )
-
-
-@probabilistic_test
-def test_compare_bootstrap_small_sample_point_estimate_performance(
```

Comment:
This test only serves to compare RMSE between bootstrap methods. This logic will be re-created in ML Flow once the legacy methods are migrated to that platform.